### ISSUE TITLE
Let hidden layers - stay hidden when moving objects.

### ISSUE THREAD

**Issue by sverrewl** [vpinball/vpinball#49]
First, Thank you for the improvements to the element selector lately !

Some vpx tables are very "crowded", since vp for a long time only supported 11 layers. I see that there is already a feature request https://github.com/vpinball/vpinball/issues/39 that can solve what frustrates me often. 

I want to create new layer and move elements to the new one. Problem is, visually nothing happens when you do this, because if the new selected layer is hidden, and you say that you want to move a object, it will automatically turn on visibility for that layer as you move it.

If the layer would stay hidden, and not change status when you move, it would give you a visual confirmation that the element was moved. Using the element selector, would be even more effective. But, maybe harder to implement ?

**Comment by iaakki** [comment]
Yes this is exactly why I came here. I've been doing a huge realignment for one project and it would have been close to impossible using 10.7, because how layers work. So I reverted to 10.6 and organizing the objects and layers it just easier. There are 2 issues. 

1. is this that the target layer wont stay hidden. It is common that one "digs" into pile of objects and then one moves wanted objects onto layer that is empty on that area. Once you assign selected objects to new layer, it would be needed them to vanish from the view. Currently the object wont disappear and that is causing a lot of mouse work. This is exactly what this issue is about.
2. issue is that the assign button is rather small and far from the actual object. Yet more mouse work and one needs to select the target layer also from bottom right box. This would not change by this issue ticket.
--> Solution for this 2nd. Could we have that 10.6 "Assign to.." menu back? That works so well. One can keep shift pressed and select the desired objects, right click on some of the selected objects (this fails often), then select assign to layer 11 and all just disappears!
![image](https://user-images.githubusercontent.com/86254776/165567881-76cf2f4f-6cae-4bff-9020-92acac33b260.png)
